haskell-src/Concordium/Types/Queries.hs

```diff
@@ -36,10 +37,10 @@ data ConsensusStatus = ConsensusStatus
       csEpochDuration :: !Duration,
       -- |Hash of the last finalized block
       csLastFinalizedBlock :: !BlockHash,
-      -- |Height of the best block (since latest regenesis)
-      csBestBlockHeight :: !BlockHeight,
-      -- |Height of the last finalized block (since latest regenesis)
-      csLastFinalizedBlockHeight :: !BlockHeight,
+      -- |Absolute height of the best block
+      csBestBlockHeight :: !AbsoluteBlockHeight,
+      -- |Absolute height of the last finalized block
+      csLastFinalizedBlockHeight :: !AbsoluteBlockHeight,
```

Comment:
I should have mentioned this in the previous round, but it would be nice to include the relative height in consensus status as well, for symmetry with block info.

Reply:
Alternatively, we could include the absolute block height of the genesis block of the current era, and the relative heights can be derived. I will leave it out for now as I don't think it's particularly valuable, though we can add it in future if there's a compelling use case.